### PR TITLE
Local and cluster configs - overrides

### LINKED ISSUE #629: Prepare configurations for minikube and cluster
We need two YAML files with configuration of overrides:
- local installation (minikube)
- cluster installation

Each YAML file may contain multiple K8s objects (configMaps/secrets etc)hh.

Acceptance Criteria:
- For local installation scripts choose only local installation overrides file
- For cluster installation scripts choose only cluster installation overrides file
- Overlapping entries are allowed (e.g. domain)

### PR DESCRIPTION
**Pull request by crabtree**
**Description**

Changes proposed in this pull request:

- Split local and cluster config into separate files

**Related issue(s)**
Resolves #629

### CODE REVIEW

File: installation/resources/installer-config-cluster.yaml.tpl [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  labels:
+    installer: overrides
+data:
+  global.isLocalEnv: "false"
+  global.domainName: "__DOMAIN__"
+  global.etcdBackup.containerName: "__ETCD_BACKUP_ABS_CONTAINER_NAME__"
+  global.etcdBackup.enabled: "__ENABLE_ETCD_BACKUP__"
+  global.alertTools.credentials.slack.apiurl: "__SLACK_API_URL_VALUE__"
+  global.alertTools.credentials.slack.channel: "__SLACK_CHANNEL_VALUE__"
+  global.alertTools.credentials.victorOps.routingkey: "__VICTOR_OPS_API_KEY_VALUE__"
```
Comment (Tomasz-Smelcerz-SAP) [inline installation/resources/installer-config-cluster.yaml.tpl]: "_ROUTING_KEY_"

File: installation/resources/installer-config-local.yaml.tpl [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ui-test-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+type: Opaque
+data:
+  test.auth.username: "YWRtaW5Aa3ltYS5jeA==" # admin@kyma.cx
```
Comment (Tomasz-Smelcerz-SAP) [inline installation/resources/installer-config-local.yaml.tpl]: Remove comments

File: installation/resources/installer-config-cluster.yaml.tpl (hunk 1 above)
Comment (crabtree) [reply installation/resources/installer-config-cluster.yaml.tpl]: Fixed

File: installation/resources/installer-config-local.yaml.tpl (hunk 2 above)
Comment (crabtree) [reply installation/resources/installer-config-local.yaml.tpl]: Removed

**Pull request merged**